src/LanguageServer.ts

```diff
@@ -230,14 +230,18 @@ export class LanguageServer {
     /**
      * Ask the client for the list of `files.exclude` patterns. Useful when determining if we should process a file
      */
-    private async getWorkspaceExcludeGlobs(workspaceFolder: string) {
-        //get any `files.exclude` globs to use to filter
-        let config = await this.connection.workspace.getConfiguration({
-            scopeUri: workspaceFolder,
-            section: 'files'
-        }) as {
-            exclude: Record<string, boolean>;
+    private async getWorkspaceExcludeGlobs(workspaceFolder: string): Promise<string[]> {
+        let config = {
+            exclude: {} as Record<string, boolean>
         };
+        //if supported, ask vscode for the `files.exclude` configuration
```

Comment:
Shouldn't this comment use the generic "client" instead of specifically mentioning "vscode"?

Reply:
The `files` and `files.exclude` are specific to VSCode. I have no idea what other editors to do handle including/excluding files from the workspace, but they're _probably_ doing it differently than this. So, the comment still makes sense, and we can enhance the comment in the future as we add support for other editors.

Reply:
That's true that the specific configuration options are not part of the standard. Although there are other clients such as Theia that model their settings after VS Code.